packages/x-data-grid/src/hooks/features/filter/gridFilterSelector.ts

```diff
@@ -96,6 +106,40 @@ export const gridFilteredSortedRowIdsSelector = createSelectorMemoized(
   (filteredSortedRowEntries) => filteredSortedRowEntries.map((row) => row.id),
 );
 
+/**
+ * Get the row ids accessible after the filtering process to position in the current tree level lookup.
+ * Does not contain the collapsed children.
+ * @category Filtering
+ */
+export const gridExpandedSortedRowIdsLookupSelector = createSelectorMemoized(
```

Comment:
So IIUC, this one returns a count? The name sounds like it returns row ids, imho.

Reply:
And do we need to make this one public or could we `@ignore` it?

Reply:
As no one asked for it, I will add `@ignore`
It returns a lookup with a position (index) in the current tree level (after sorting and filtering)
I will rename it to `gridExpandedSortedRowTreeLevelPositionLookupSelector`